From 2e5e9e16a4847b2e60cabc70e6b9bd7d633cb68a Mon Sep 17 00:00:00 2001
From: Christos Zoulas <christos@zoulas.com>
Date: Sat, 22 Jul 2017 17:21:24 +0000
Subject: [PATCH] Strength decreased in magic file magic/Magdir/dump for new-fs
 dump file (little endian) to correctly recognize '*.mo' GNU message catalog
 (little endian).

https://bugzilla.redhat.com/show_bug.cgi?id=1226215

Upstream commit: https://github.com/file/file/commit/2e5e9e16a4847b2e60cabc70e6b9bd7d633cb68a
---
 magic/Magdir/dump | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/magic/Magdir/dump b/magic/Magdir/dump
index 64742b7a..c5711661 100644
--- a/magic/Magdir/dump
+++ b/magic/Magdir/dump
 >0	use	old-dump-be
 
 24	lelong	60012		new-fs dump file (little endian),
+# to correctly recognize '*.mo' GNU message catalog (little endian)
+!:strength - 15
 >0	use	\^new-dump-be
 
 24	lelong	60011		old-fs dump file (little endian),
-- 
2.13.3

